extensions/amp-sidebar/0.2/amp-sidebar.js

```diff
@@ -632,7 +632,7 @@ export class AmpSidebar extends AMP.BaseElement {
   getMaskElement_() {
     if (!this.maskElement_) {
       const mask = this.document_.createElement('div');
-      mask.classList.add('i-amphtml-sidebar-mask');
+      mask.classList.add('amphtml-sidebar-mask', 'i-amphtml-sidebar-mask');
```

Comment:
and here
```suggestion
      mask.classList.add('amp-sidebar-mask');
```

Reply:
Keeping both names to prevent a larger scope.